src/components/UserInfo.vue

```diff
@@ -56,31 +56,38 @@
             v-on:change="getUserClientRoles()"
           ></v-autocomplete>
         </v-col>
-        <v-col class="col-7">
-          <label v-show="selectedClientId">Roles</label>
-          <div class="checkbox-group" v-if="selectedClientId">
-            <v-checkbox
-              hide-details="auto"
-              v-for="role in effectiveClientRoles"
-              v-model="selectedRoles"
-              :value="role"
-              :label="role.name"
-              v-bind:key="role.name"
-            ></v-checkbox>
-            <v-checkbox
-              hide-details="auto"
-              v-for="role in availableClientRoles"
-              v-model="selectedRoles"
-              :value="role"
-              :label="role.name"
-              v-bind:key="role.name"
```

Comment:
`:` is short for v-bind, and `@` is short for `v-on` I think. Do you have a preference for whether we use the shorthand of not? As a beginner I kind of like reading `v-on` but I'd get use to `@` fast enough.